web/Views/Admin/PropertyList.cshtml

```diff
@@ -93,18 +107,19 @@
 @{ var from = (Model.Page - 1) * Model.PropPerPage + 1;
     var to = Math.Min((Model.Page - 1) * Model.PropPerPage + Model.PropPerPage, Model.Total);}
 
-<p>Displaying properties @from to @to of @Model.Total</p>
+@if (Model.Properties.Count > 0)
+{<p>Displaying properties @from to @to of @Model.Total</p>}
 
 @if (Model.Page > 1)

```

Comment:
Could these next and previous page button be extracted into a component?

Reply:
I've extracted the 'Displaying [items] [min] to [max] of [total]' code, but the next and previous page buttons are awkward (there are different values being passed in depending on the page - sometimes a sortBy, sometimes a filter, sometimes a postcode).

Reply:
From what it looks like, could the pagination model just take in a previousHref and nextHref? you can still define the links in their relative files using the Url.Action, but wouldn't need all the html etc

Reply:
Yes, that would work I suppose.